code/modules/research/server.dm

```diff
@@ -1,390 +1,147 @@
+// These currently dont actually do anything
+// They will serve a purpose once the R&D rework is done
+
 /obj/machinery/r_n_d/server
 	name = "R&D Server"
```

Comment:
```suggestion
	name = "\improper R&D Server"
```
I'm guessing there's no easy way for us to make it say "a**n** R&D Server" instead of "a R&D Server".